app/assets/javascripts/discourse/app/initializers/post-decorations.js

```diff
@@ -176,8 +177,12 @@ export default {
 
           const popupBtn = _createButton();
           table.parentNode.classList.add("fullscreen-table-wrapper");
-          table.parentNode.insertBefore(popupBtn, table);
+          // Create a button wrapper for case of multiple buttons (i.e. table builder extension)
+          const buttonWrapper = document.createElement("div");
+          buttonWrapper.classList.add("fullscreen-table-wrapper-buttons");
```

Comment:
```suggestion
          buttonWrapper.classList.add("fullscreen-table-wrapper--buttons");
```

Super-nitpicky, but slightly more BEMy class name with the two dashes.